src/components/file-upload.tsx

```diff
@@ -155,6 +156,7 @@ export class GenerateTable extends React.Component<GenerateTableProps> {
     } else if (manual) {
       tableRows = Object.keys(manualTable).map((year, key) => {
         const earningValue = manualTable[year]
+        /* Can this be parseable with symbols $, */
```

Comment:
Oops, is this a question for your next version or something that should go into the code?
```suggestion
```